src/mapml/utils/Util.js

```diff
@@ -122,15 +125,25 @@ export var Util = {
         default:
       }
     }
+    if (numberOfAxes >= 2) {
+      locInputs = true;
+    }
     let zoomBoundsFormatted = {
       minZoom:+template.zoomBounds.min,
       maxZoom:+template.zoomBounds.max,
       minNativeZoom:nMinZoom,
       maxNativeZoom:nMaxZoom
     };
+    if(!locInputs && template.extentPCRSFallback && template.extentPCRSFallback.bounds) {
```

Comment:
I test for the existence of the template.extentPCRSFallback and its .bounds property, so why should I have to include the object at all in the test data?